rootfs/usr/share/inputplumber/devices/50-steam_deck.yaml

```diff
@@ -34,16 +34,16 @@ source_devices:
       product_id: 0x1205
       interface_num: 2
   # Touchscreen
-  - group: mouse
-    hidraw:
-      vendor_id: 0x2808
-      product_id: 0x1015
-  - group: mouse
+  - group: touchscreen
     unique: false
-    blocked: true
     evdev:
       name: "FTS3528:00 2808:1015*"
       handler: event*
+    config:
+      touchscreen:
+        orientation: "right"
+        width: 1280
```

Comment:
I think it might be better to use an aspect ratio here and scale it based in the true resolution. We would need to detect that

Reply:
I left `width` and `height` here to allow users to override the width/height of the virtual touchscreen that gets created, but I have updated this PR to now query the source touchscreen for its size if these are not defined.